content/blog/dotnet-preview/index.md

```diff
@@ -0,0 +1,232 @@
+   $ pulumi stack output endpoint
+   https://azpulumi-as0ef47193.azurewebsites.net
+   $ curl "$(pulumi stack output endpoint)"
+   <html>
+      <body>
+         <h1>Greetings from Azure App Service!</h1>
+      </body>
+   </html>
+   ```
+
+From there, feel free to experiment. Simply making edits and running `pulumi up` will incrementally update your stack. Once finished, tear down your stack's resources by destroying and removing it:
+
+   ``` bash
+   $ pulumi destroy --yes
+   $ pulumi stack rm --yes
+   ```
+
+## Recap
+
+In this post we've shown how to use Pulumi to deploy a modern application to Azure. The example code performs a number of tasks.
+
+First, we created a [resource group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-overview#resource-groups), a [storage account](https://docs.microsoft.com/en-us/azure/storage/common/storage-redundancy-lrs), and an [application service plan](https://docs.microsoft.com/en-us/azure/app-service/overview-hosting-plans) for our application:
+
+   ```C#
+   var resourceGroup = new ResourceGroup("appservice-rg"); 
+
+   var storageAccount = new Account("sa", new AccountArgs
+   {
+
+       ResourceGroupName = resourceGroup.Name,
+       AccountReplicationType = "LRS",
+       AccountTier = "Standard",
+
+   }); 
+
+   var appServicePlan = new Plan("asp", new PlanArgs
+   {
+
+       ResourceGroupName = resourceGroup.Name,
+       Kind = "App",
+       Sku = new PlanSkuArgs
+       {
+           Tier = "Basic",
+           Size = "B1",
+       },
+
+   });
+   ``` 
+
+We then package the `wwwroot` directory containing the `index.html` page into the storage account we created previously.
+
+   ```C#
+   var container = new Container("zips", new ContainerArgs
+   {
+       StorageAccountName = storageAccount.Name,
+       ContainerAccessType = "private",
+   });
+
+   var blob = new ZipBlob("zip", new ZipBlobArgs
+   {
+       StorageAccountName = storageAccount.Name,
+       StorageContainerName = container.Name,
+       Type = "block",
+       Content = new FileArchive("wwwroot"),
+   });
+
+   var codeBlobUrl = SharedAccessSignature.SignedBlobReadUrl(blob, storageAccount);
+   ```
+
+Using the configuration we set with the Pulumi CLI, the program creates a SQLServer deployment.
+
+   ```C#
+   var config = new Config();
+   var username = config.Get("sqlAdmin") ?? "pulumi";
+   var password = config.RequireSecret("sqlPassword");
+   var sqlServer = new SqlServer("sql", new SqlServerArgs
+   {
+       ResourceGroupName = resourceGroup.Name,
+       AdministratorLogin = username,
+       AdministratorLoginPassword = password,
+       Version = "12.0",
+   });
+
+   var database = new Database("db", new DatabaseArgs
+   {
+       ResourceGroupName = resourceGroup.Name,
+       ServerName = sqlServer.Name,
+       RequestedServiceObjectiveName = "S0",
+   });
+   ```
+
+We deploy the app service which includes the website and the connection to the SQLServer instance.
```

Comment:
Don't we want to title-case service names, e.g., App Service?

Reply:
yes, accepted